cdap-ui/app/directives/my-pipeline-summary/my-pipeline-summary.html

```diff
@@ -35,8 +35,13 @@
         <td>{{MyPipelineSummaryCtrl.totalRunsCount}}</td>
         <td>{{MyPipelineSummaryCtrl.avgRunTime}}</td>
         <td>
-          {{ MyPipelineSummaryCtrl.nextRunTime | amDateFormat:'h:mm:ss A'}}
-          <small> {{ MyPipelineSummaryCtrl.nextRunTime | amDateFormat:'MM/DD/YYYY'}}</small>
+          <span ng-if="MyPipelineSummaryCtrl.nextRunTime !== '--'">
```

Comment:
I don't think testing against a magic string such as this is a good idea. Wouldn't an empty string be a better idea as the meaning of no next run time?